assets/src/dashboard/dataUtils/formattedStoriesArray.js

```diff
@@ -249,7 +249,8 @@ const formattedStoriesArray = [
         ],
         'wp:attachment': [
           {
-            href: 'http://localhost:8899/wp-json/wp/v2/media?parent=167',
+            href:
+              'http://localhost:8899/wp-json/web-stories/v1/media?parent=167',
```

Comment:
Can you explain why this change is needed?

Reply:
@diegovar The REST API URL in WordPress changed from including `/wp/v2` to `/web-stories/v1`. This change here merely reflects that. The rest is Prettier.